Sources/StringExtensions.swift

```diff
@@ -402,7 +402,7 @@ extension String {
             paragraphStyle.lineBreakMode = lineBreakMode!
             attrib.updateValue(paragraphStyle, forKey: NSParagraphStyleAttributeName)
         }
-        let size = CGSize(width: width, height: CGFloat(DBL_MAX))
+        let size = CGSize(width: width, height: CGFloat(Float.greatestFiniteMagnitude))
```

Comment:
Just checked, `Double.greatestFiniteMagnitude` is the same, just committed a change. Thanks.